### PR TITLE
fix "Choices" regexp

### PR DESCRIPTION
**Pull request by AsaiKen**
`|css` matches all scriptSrc including "css". It is not correct. I fix it.

### CODE REVIEW

File: src/technologies/c.json [hunk 1]
```diff
@@ -1361,7 +1361,7 @@
     "icon": "Choices.png",
     "oss": true,
     "scriptSrc": [
-      "choices\\.js/|@([\\d.]+)(?:/assets)?(?:/scripts)?(?:/styles)?(?:/dist)?/choices(?:\\.min)?\\.js|css\\;version:\\1"
```
Comment (nurbek91) [inline src/technologies/c.json:1364]: I'm afraid this **scriptSrc** attribute only checks **js** files.
Could you provide a link where this technology is not defined.
Apparently we need to add the **dom** attribute and check the **css** file

Comment (AsaiKen) [reply src/technologies/c.json:1364]: This rule detects at "https://www.aeyesec.jp/" by scriptSrc "https://static.parastorage.com/services/communities-blog-ooi/1.546.0/PostListViewerWidgetNoCss.bundle.min.js"
It seems false-positive.

Comment (nurbek91) [reply src/technologies/c.json:1364]: Strange, but I do not see this technology.
Could you delete the cached results and look again

Comment (AsaiKen) [reply src/technologies/c.json:1364]: Is this procedure for removing the cache okay?
https://user-images.githubusercontent.com/12981948/204837007-43ce6984-8b92-403c-9bb0-8c7deab830b3.mp4

Comment (nurbek91) [reply src/technologies/c.json:1364]: Ah, ok i will take a look

Comment (nurbek91) [reply src/technologies/c.json:1364]: ###  Hey mate, can u replace code with this one.

```
 "js": {
      "Choices": ""
    },
  "scriptSrc": [
      "choices\\.js(?:@|/)?([\\d\\.]+)?.+choices\\.min\\.js\\;version:\\1",
      "/modules/choices/js/choices\\.js"
    ],
```

###  Add new detection examples: 
http://grapp.ir/
https://www.vashdosug.ru/
https://portal.ciee.org.br/
https://kvf.fo/
http://www.designindaba.com/

### remove false-positive example:
https://www.aeyesec.jp/

Comment (AsaiKen) [reply src/technologies/c.json:1364]: oh, it works fine.

**Pull request closed without merging**